packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -582,152 +583,161 @@ describe('Launchpad: Setup Project', () => {
                   .find('svg')
-                  .should('have.attr', 'data-cy', `${Cypress._.lowerCase(bundler.name)}-logo`)
-                  .click()
+                  .should('have.attr', 'data-cy', `${Cypress._.lowerCase(supportedBundler)}-logo`)
+                })
+
+                cy.findByRole('option', { name: bundler.name })
+                .find('svg')
+                .should('have.attr', 'data-cy', `${Cypress._.lowerCase(bundler.name)}-logo`)
+                .click()
 
-                  cy.findByRole('button', { name: `Bundler(Dev Server) ${bundler.name}` }) // ensure selected option updates
-                }
+                cy.findByRole('button', { name: `Bundler(Dev Server) ${bundler.name}` }) // ensure selected option updates
+              }
 
-                cy.findByRole('button', { name: lang.name }).click()
+              cy.findByRole('button', { name: lang.name }).click()
 
-                cy.log('Go to next step')
-                cy.get('@nextStepButton').should('not.have.disabled').click()
+              cy.log('Go to next step')
+              cy.get('@nextStepButton').should('not.have.disabled').click()
 
-                cy.contains('h1', 'Install Dev Dependencies')
-                cy.contains('p', 'Paste the command below into your terminal to install the required packages.')
+              cy.contains('h1', 'Install Dev Dependencies')
+              cy.contains('p', 'Paste the command below into your terminal to install the required packages.')
 
-                cy.log('Return to previous step')
-                cy.findByRole('button', { name: 'Back' })
-                .click()
+              cy.log('Return to previous step')
+              cy.findByRole('button', { name: 'Back' })
+              .click()
 
-                cy.findByRole('button', { name: `Front-end Framework ${framework.name}` })
-                if (framework.supportedBundlers.length > 1) {
-                  cy.findByRole('button', { name: `Bundler(Dev Server) ${bundler.name}` })
-                }
+              cy.findByRole('button', { name: `Front-end Framework ${framework.name}` })
+              if (framework.supportedBundlers.length > 1) {
+                cy.findByRole('button', { name: `Bundler(Dev Server) ${bundler.name}` })
+              }
 
-                cy.findByRole('button', { name: lang.name })
-                cy.findByRole('button', { name: 'Next Step' }).click()
+              cy.findByRole('button', { name: lang.name })
+              cy.findByRole('button', { name: 'Next Step' }).click()
 
-                cy.log('Go to next step and verify Install Dev Dependencies page')
-                cy.contains('h1', 'Install Dev Dependencies')
+              cy.log('Go to next step and verify Install Dev Dependencies page')
+              cy.contains('h1', 'Install Dev Dependencies')
 
-                let installCommand = `npm install -D ${framework.package} ${bundler.package}`
```

Comment:
I put more real-world tests for this here https://github.com/cypress-io/cypress/pull/20306/files#diff-4bac4d0f514a75ab43c003764635a9da4f8bb8f7d8e9300d9f2abe353fa08dde that actually use real projects. They also run much faster and are closer to the actual code they test.